app/src/main/java/com/mobilecoronatracker/ui/countrieslist/FragmentCountriesList.kt

```diff
@@ -95,5 +96,11 @@ class FragmentCountriesList : Fragment() {
             adapter.countriesReports = it
             adapter.notifyDataSetChanged()
         })
+        viewModel.navigationToCountryRequested.observe(viewLifecycleOwner, Observer {
+            if (it.isBlank()) return@Observer
```

Comment:
I would prefer brackets when it is not if else inline statement

Reply:
done